api/api.yaml

```diff
@@ -112,6 +94,31 @@ paths:
         '500':
           $ref: '../result/request.yaml#/components/responses/InternalServerError'
 
+  /control/flows:
+    post:
+      tags: ['Control']
+      operationId: update_flow_state
```

Comment:
flow state may be a little ambiguous; maybe we should just call it `update_flows`